specs/packagemanager/PackageManagement.md

```diff
@@ -857,6 +971,41 @@ namespace Microsoft.Windows.Management.Deployment
         NewerAvailable       = 2,
     };
 
+    [contract(PackageDeploymentContract, 3)]
+    [default_interface]
+    runtimeclass AppxPackagingObject{
```

Comment:
Oddly generic name ('Object'). I can't think of a good name atm but that's OK. This makes for a good focal point for API Review. I'm sure there'll be feedback and 'vigorous discussion' :P